doc/source/examples/sharp-immersed-boundary/flow-around-particle-sharp/flow-around-particle-sharp.rst

```diff
@@ -0,0 +1,209 @@
+Flow around a Finite Cylindrical Particle Using the Sharp Interface Method
+==============================================================================
+
+.. This example focuses on the drag coefficient validation for a perpendicular (theta = 90) cylindrical particle for low Reynolds [0.1, 1] and an aspect ratio of 2. 
+.. A bigger parameter sweep could be documented in a future example.
+
+----------------------------------
+Features
+----------------------------------
+
+- Solvers: ``lethe-fluid-sharp`` (with Q1-Q1) 
+- Steady-state problem
+- Non-spherical particle in resolved flow
+- Force calculation validation
+
+---------------------------
+Files Used in This Example
+---------------------------
+
+- Parameter file: ``examples/sharp-immersed-boundary/flow-around-particle-sharp/flow-around-particle-sharp.prm``
+
+------------------------
+Description of the Case
+------------------------
+
+This example showcases the use of sharp immersed boundary conditions to model geometries of non-spherical shapes in flow, in this case a cylindrical particle perpendicular to the flow. 
+This example focuses on small cylindrical particles (:math:`Ar = L/D = 2`) at low Reynolds numbers ranging from 0.1 to 1. To validate the simulation results, the derived drag coefficient is 
+compared to the expected results from the litterature. 
+
+.. image:: images/flow-around-particle-sharp-case.png
+    :alt: Simulation schematic
+    :align: center
+
+---------------
+Parameter File
+---------------
+
+Mesh
+~~~~
+
+The geometry, as shown in the previous figure, is fully defined by the particle's geometry. The mesh is first defined coarsely away from the particle with elements of size :math:`L \times L \times L` with :math:`L = 2`. 
+We then make use of a ``box refinement`` to refine the mesh around the particle and properly evaluate the flow field and the wake with elements of size :math:`0.25D \times 0.25D \times 0.25D` with :math:`D = 1`.
+
+.. code-block:: text 
+
+    subsection mesh
+        set type                   = dealii
+            set grid type          = subdivided_hyper_rectangle
+            set grid arguments     = 2,1,1 : -24,-16,-16 : 40,16,16 : true
+            set initial refinement = 4
+        end
+
+    subsection box refinement 
+        set initial refinement = 3
+        subsection mesh
+            set type               = dealii
+            set grid type          = subdivided_hyper_rectangle
+            set grid arguments     = 2,1,1: -6,-4,-4 : 10,4,4 : true
+            set initial refinement = 0
+        end
+    end
+
+IB Particles
+~~~~~~~~~~~~~
+
+We define the particle in flow with the use of the sharp immersed boundary method. The particle is defined as a cylinder with a radius of 0.5 and a half-length of 1.0. The particle is centered at (0,0) 
+and oriented perpendicularly to the flow. The ``local mesh refinement`` is set to 3 to ensure a fine mesh and properly resolve the boundary layer flow around the particle.
```

Comment:
What is perpendicular to what here?

Reply:
The cylinder is perpendicular to the flow, maybe it will be clearer with the schematic now. I restate it here as we orient the particle in the prm file in order to make it perpendicular.